File: src/main/java/dqualizer/research/dqapi/repositories/dam/domainstory/WorkObjectRepository.java
package dqualizer.research.dqapi.repositories.dam.domainstory;

import io.github.dqualizer.dqlang.types.dam.domainstory.WorkObject;
import org.springframework.data.mongodb.repository.MongoRepository;

public interface WorkObjectRepository extends MongoRepository<WorkObject, String> {
}

File: src/main/kotlin/io/github/dqualizer/dqapi/repositories/dam/domainstory/WorkObjectRepository.kt
package io.github.dqualizer.dqapi.repositories.dam.domainstory

import io.github.dqualizer.dqlang.types.dam.domainstory.WorkObject
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository

@Repository
interface WorkObjectRepository : MongoRepository<WorkObject, String>
